src/lib/user/user.store.ts

```diff
@@ -1,16 +1,18 @@
 import { UserManager, type ConfigOptions } from '@forgerock/javascript-sdk';
 import { writable, type Writable } from 'svelte/store';
 
+import type { Maybe } from '$lib/interfaces';
+
 export interface UserStore extends Pick<Writable<UserStoreValue>, 'subscribe'> {
   get: (getOptions?: ConfigOptions) => void;
   reset: () => void;
 }
 export interface UserStoreValue {
   completed: boolean;
-  error: {
-    code?: number | null;
-    message: string | null;
-  } | null;
+  error: Maybe<{
```

Comment:
I'm seeing `Maybe` pop up quite often in this PR - I had to look up what the Maybe pattern is, and now I think I understand. Is there any tangible difference to the way we handle nulls etc by using this pattern, or is it just for reducing duplication and increasing consistency within the app?

Reply:
More the latter. Just reduces the `thing | null | undefined` littered all over the place.